.autoblocks.yml

```diff
@@ -4,9 +4,13 @@ autogenerate:
 
     prompts:
       - id: used-by-ci-dont-delete
-        major_versions:
-          - 2
+        major_version: 2
 
       - id: used-by-ci-dont-delete-no-params
-        major_versions:
```

Comment:
switching from a list to a single minor version. i think supporting a list here was overly complicated and an unlikely edge case. can reconsider if someone actually wants to use multiple major versions of the same prompt simultaneously